contracts/AelinPool.sol

```diff
@@ -228,6 +239,9 @@ contract AelinPool is AelinERC20, MinimalProxyFactory, IAelinPool, ReentrancyGua
         //The running total for 721 tokens
         uint256 maxPurchaseTokenAmount;
 
+        //True => ERC721, False => ERC1155
+        bool nftType;
```

Comment:
naming convention is not great here. I'd rather have `isCollectionERC721` or something similar. If false then we assume it's 1155.